libs/api/observability/src/lib/filters/sentry-exceptions.filter.ts

```diff
@@ -1,23 +1,35 @@
-import { Catch, ExceptionFilter } from '@nestjs/common';
+import { Catch, ExceptionFilter, Logger } from '@nestjs/common';
 import * as Sentry from '@sentry/node';
 
 import { PresentableException } from '@kordis/api/shared';
 
+import { KordisLogger } from '../services/kordis-logger.interface';
+
 @Catch()
 export class SentryExceptionsFilter implements ExceptionFilter {
+	readonly logger: KordisLogger;
+
+	constructor(_logger: Logger) {
+		this.logger = _logger;
+	}
+
 	catch(exception: unknown): void {
 		if (exception instanceof PresentableException) {
 			// if this is a presentable error, such as a validation error, we don't want to log it as an error but rather as an information to have the context for possible future debugging
-			Sentry.addBreadcrumb({
-				level: 'error',
-				message: exception.message,
-				data: {
-					code: exception.code,
-					name: exception.name,
-					stack: exception.stack,
-				},
+			this.logger.error(exception.message, undefined, {
```

Comment:
```suggestion
			this.logger.info(exception.message, undefined, {
```

Shouldn't this be `.info` to log for debug purpose and not as error?

Reply:
changed to log type info: https://github.com/kordis-leitstelle/kordis/pull/653/commits/353eee9b47760a6a14a83efde016c0da9924539d